ngui/ui/src/theme.ts

```diff
@@ -463,6 +509,12 @@ const getThemeConfig = (settings = {}) => {
               padding: MPT_SPACING_3,
               borderRadius: MPT_SPACING_1
             }
+          },
+          ".MuiBox-WhiteCard": {
+            boxShadow: MPT_BOX_SHADOW,
+            background: "#FFFFFF",
```

Comment:
```suggestion
            background: MPT_BRAND_WHITE,
```

plus add import of course

Reply:
Fixed